code/game/turfs/change_turf.dm

```diff
@@ -105,8 +107,24 @@ GLOBAL_LIST_INIT(blacklisted_automated_baseturfs, typecacheof(list(
 	//We do this here so anything that doesn't want to persist can clear itself
 	var/list/old_comp_lookup = comp_lookup?.Copy()
 	var/list/old_signal_procs = signal_procs?.Copy()
+
+	var/old_atoms_state = SSatoms.initialized
+	SSatoms.initialized = INITIALIZATION_INSSATOMS
```

Comment:
Is this a regression? Why is this here?

Reply:
Because if you change turf during mapload, the turf that was changed will never be initialised since it is created but is either never added to the atom initialisation list or may be added twice in some cases due to turf references transfering to the new turf. This will just force the turf to always initialise instantly when calling changeturf. I can't exactly remember quite why it was needed, but I think it was because z-mimic checks for the initialisation state before setting up z-mimic and the atom was being initialised while SSatoms was in an incorrect mode. I still can't remember exactly why it was needed, but I can take a look at it again in code and see what the problem was without it

Reply:
I'm worried this will break some niche types of maploading